### PR TITLE
Tickets/dm 36020

### PR DESCRIPTION
**Pull request by teweitsai**
* Support the EUI specifc items:

  * Switch digital output command.
  * Configuration event.
  * Tangent force error telemetry.
  * Telescope mount assembly (TMA) inclinometer angle telemetry.
  * Raw power status telemetry.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -68,7 +68,7 @@ pipeline {
                         export TS_CONFIG_MTTCS_DIR=${WORK_HOME}/ts_config_mttcs
 
                         setup -k -r .
-                        pytest --cov-report html --cov=${env.MODULE_NAME} --junitxml=${env.XML_REPORT}
+                        pytest tests/ --cov-report html --cov=${env.MODULE_NAME} --junitxml=${env.XML_REPORT}
```
Comment (tribeiro) [inline Jenkinsfile:71]: Why are you restricting `pytest` to run only on the `tests` folder? this excludes checking flake8, black, etc from the entire package. I know we are in the process of removing the plugins but even then, I still think it is best is you just run `pytest` on everything.

File: tests/test_controller_eui.py [hunk 2]
```diff
@@ -564,6 +564,52 @@ async def test_set_mirror_home(self):
...
+            server
+        ) as controller:
+
+            # Switch the communication power
+            await controller.write_command_to_server(
+                "switchDigitalOutput",
+                message_details={"bit": DigitalOutput.CommunicationPower.value},
+            )
+
+            await asyncio.sleep(1)
```
Comment (tribeiro) [inline tests/test_controller_eui.py:589]: Consider extracting the 1 into a variable/constant.

File: tests/test_controller_eui.py [hunk 3]
```diff
@@ -564,6 +564,52 @@ async def test_set_mirror_home(self):
...
+            )
+            self.assertTrue(server.model.communication_power_on)
+
+            # Switch the motor power
+            await controller.write_command_to_server(
+                "switchDigitalOutput",
+                message_details={"bit": DigitalOutput.MotorPower.value},
+            )
+
+            await asyncio.sleep(1)
```
Comment (tribeiro) [inline tests/test_controller_eui.py:605]: Same as above, consider extracting the sleep time into a variable/constant

File: tests/mock/test_mock_model.py [hunk 4]
```diff
@@ -238,6 +239,25 @@ def test_get_ilc_data(self):
...
+    def test_calculate_force_error_tangent(self):
+
+        self.model.control_open_loop.inclinometer_angle = 89.853
+        tangent_force_current = np.array(
+            [-325.307, -447.377, 1128.37, -1249.98, 458.63, 267.627]
+        )
+        data = self.model._calculate_force_error_tangent(tangent_force_current)
+
+        force_error_tangent = data["force"]
+        self.assertEqual(force_error_tangent[0], -325.307)
```
Comment (tribeiro) [inline tests/mock/test_mock_model.py:251]: Here and in the following assertions, you are using hard coded values. Consider extracting these into variables and unifying them in a single location you could easily update afterwards. 

Something like a method:

```
def get_expected_force_error_tangent(self):
    return ...
```

then here you could do:

```
expected_force_error_tangent, expected_force_error_tangent_tolerance = self.get_expected_force_error_tangent()

for value, expected, tolerance in zip(force_error_tangent, expected_force_error_tangent, expected_force_error_tangent_tolerance):
    if tolerance > 0.0:
        self.assertAlmostEqual(value, expected, tolerance)
    else:
        self.assertEqual(value, expected)
```

File: tests/mock/test_mock_model.py [hunk 5]
```diff
@@ -238,6 +239,25 @@ def test_get_ilc_data(self):
...
+        self.assertAlmostEqual(force_error_tangent[1], -377.4539166)
+        self.assertAlmostEqual(force_error_tangent[2], 987.1830153)
+        self.assertEqual(force_error_tangent[3], -1249.98)
+        self.assertAlmostEqual(force_error_tangent[4], 387.1993005)
+        self.assertAlmostEqual(force_error_tangent[5], 221.7858503)
+
+        self.assertAlmostEqual(data["weight"], -0.743948)
```
Comment (tribeiro) [inline tests/mock/test_mock_model.py:258]: Same as above, consider extracting the comparison values to some variable

File: Jenkinsfile (hunk 1 above)
Comment (teweitsai) [reply Jenkinsfile]: Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!

File: tests/test_controller_eui.py (hunk 2 above)
Comment (teweitsai) [reply tests/test_controller_eui.py]: Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!

File: tests/test_controller_eui.py (hunk 3 above)
Comment (teweitsai) [reply tests/test_controller_eui.py]: Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!

File: tests/mock/test_mock_model.py (hunk 4 above)
Comment (teweitsai) [reply tests/mock/test_mock_model.py]: Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!

**Pull request merged**